Automattic/wp-calypso
# Theme Showcase: Remove unused CSS code

Related to #75457

## Proposed Changes

This PR removes unused CSS code related to #75422, which implements a new component for theme cards.

## Testing Instructions



* Head to the Theme Showcase.
* Ensure that the theme card UI is the same as the one in production.
* Please test in Simple and Atomic sites.
* Also test in the logged-out Theme Showcase.

## Pre-merge Checklist
